### PR TITLE
main/syslog-ng: config fixes

### PR DESCRIPTION
**Pull request by uu**
- fix config header version
- fix makedeps
- disable tests due to lack of criterion
- enable hw RNG

### CODE REVIEW

File: main/syslog-ng/APKBUILD [hunk 1]
```diff
@@ -71,6 +78,7 @@ prepare() {
 build() {
 	cd "$builddir"
 
+	autoreconf -if
```
Comment (maxice8) [inline main/syslog-ng/APKBUILD]: This should be in the prepare() phase

Comment (uu) [reply main/syslog-ng/APKBUILD]: done. Thanks:)

File: main/syslog-ng/APKBUILD [hunk 2]
```diff
@@ -1,18 +1,25 @@
 # Contributor: jv <jens@eisfair.org>
 # Contributor: Adrian Guenter <adrian@gntr.me>
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
```
Comment (Cogitri) [inline main/syslog-ng/APKBUILD]: Nitpick, but new contributors usually add themselves to the top of the file.

File: main/syslog-ng/APKBUILD [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
 # Maintainer: jv <jens@eisfair.org>
 pkgname=syslog-ng
 pkgver=3.22.1
-pkgrel=1
+pkgrel=2
 pkgdesc="Next generation logging daemon"
 url="http://www.balabit.com"
 arch="all"
 license="GPL-2.0-or-later"
+options="!check" # no criterion on alpine
```
Comment (Cogitri) [inline main/syslog-ng/APKBUILD]: Not quite sure if I understand this.

Comment (uu) [reply main/syslog-ng/APKBUILD]: http://www.serpentine.com/criterion needed for the test

Comment (Cogitri) [reply main/syslog-ng/APKBUILD]: Maybe rephrase it to  "Missing dep: criterion" then to make it clearer what you mean.

Comment (uu) [reply main/syslog-ng/APKBUILD]: done. Thank you:)
